lblod/ember-rdfa-editor-lblod-plugins
# Use the new template uris on variables

### Overview
When we use the variables on the final agendapoint the uris are the same, so if we insert 2 snippets with the same variable the rdfa parser does weird things with the values

##### connected issues and PRs:
GN-5232


### Setup
None

### How to test/reproduce
create a snippet in RB with variables, insert that snippet 2 times in GN, observe that the variables have different uuids on GN even if they come from the same snippet

### Challenges/uncertainties
None



### Checks PR readiness
